roles/cnf_cert/tasks/logging.yml

```diff
@@ -67,8 +67,15 @@
 # this feature is enabled (disabled by default).
 # This check can only be done if CNF Cert was executed correctly (so claim.json file exists).
 - name: Confirm we are testing the correct CNF Cert code
+  vars:
+    log_file_name: >
+      {% if test_network_function_version is version("v5.1.0", "<=") and test_network_function_version != "HEAD" %}
```

Comment:
Can we switch it to ternary?
"{{ (test_network_function_version is version("v5.1.0", "<=") and test_network_function_version != "HEAD") | ternary('dci-tnf-execution.log', 'certsuite.log')}}"

Reply:
I started doing that in that way because I planned to use more than two values but finally I just used two alternatives, so yes, it's fine for me to move to ternary.

Reply:
I've finally kept dci-tnf-execution.log file for tnf v5.1.1 too, so I've removed this new code and I'll keep using the old one.